docs/install-existing.md

```diff
@@ -10,9 +10,9 @@ The Application Gateway Ingress controller runs as pod within the AKS cluster. I
 
 In order to install the ingress controller on AKS we use [Helm](https://docs.microsoft.com/en-us/azure/aks/kubernetes-helm).
 
-## Assumptions
```

Comment:
correct the hypelink to this section from the content table

Reply:
Good catch!  We need a markdown static analyzer to catch these things automatically. 🔜